model-optimizer/extensions/back/ConvolutionNormalizer.py

```diff
@@ -73,38 +87,14 @@ class V10ConvolutionWithGroupsResolver(BackReplacementPattern):
     """
     enabled = False
 
-    @staticmethod
-    def pattern():
-        return dict(
-            nodes=[
-                ('node', dict(type='Convolution', group=lambda g: g is not None and g != 1))
-            ],
-            edges=[]
-        )
-
-    def replace_pattern(self, graph: Graph, match: dict):
-        node = match['node']
-
-        group = node.group
-        assert group > 1
-
-        weights_shape = node.in_port(1).data.get_shape()
-        assert weights_shape is not None
-        assert weights_shape[0] % group == 0
-        I = node.in_port(0).data.get_shape()[1]
-
-        new_shape = int64_array([group, node.output / group, I / group, *weights_shape[2:]])
-
-        assert np.prod(weights_shape) == np.prod(new_shape), \
-            'Initial weights shape {}, grouped weights shape {}'.format(weights_shape, new_shape)
-
-        del node['group']
-        node['type'] = 'GroupConvolution'
-
-        reshape = create_op_node_with_second_input(graph, Reshape, int64_array(new_shape),
-                                                   {'override_output_shape': True})
-
-        node.in_port(1).get_connection().insert_node(reshape)
+    def find_and_replace_pattern(self, graph: Graph):
+        for node in graph.get_op_nodes(type='Convolution'):
+            group = node.soft_get('group', None)
+            if group is not None:
+                if group == 1 and node.soft_get('op') is not 'DepthwiseConv2dNative':
+                    continue
+                else:
+                    resolve_convolution_with_group(node, group, ir_version='V10')
```

Comment:
These lines can be optimized to:
```python
if group != 1 or node.soft_get('op') == 'DepthwiseConv2dNative':
    resolve_convolution_with_group(node, group, ir_version='V10')
```

The same comment to the transformation above